src/LogFileReader.cpp

```diff
@@ -46,11 +46,32 @@ bool LogFileReader::readAll(const QString& fileName)
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
     // TODO implement this first
+    QStringList sections = line.split(", ");
+    if(sections.length() != 3)
```

Comment:
Move the constants such as this 3 and the time format into meaningfully named constants at the top of the time.